client-test/src/test/resources/JerseyOEmbedClientFTest/vimeo.midas.com

```diff
@@ -0,0 +1,7 @@
+OEmbedResponse{type=video, version=1.0, authorName=Uncorked Productions, authorURI=https://vimeo.com/uncorkedprods, providerName=Vimeo, providerURI=https://vimeo.com/, thumbnailHeight=337, thumbnailURI=https://i.vimeocdn.com/video/605324679_640.jpg, thumbnailWidth=640, title=Maxine the Fluffy Corgi, height=338, html=<iframe src="https://player.vimeo.com/video/189789787" width="640" height="338" frameborder="0" title="Maxine the Fluffy Corgi" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>, width=640, customProperties={is_plus=0, duration=34, description=Short legs. Big city.
```

Comment:
So midas is suppose to not change the extension if you have foobar.jpg it would be foobar.midas.jpg however its misreading .com as an extension. Any ideas?